src/main/java/frc/robot/subsystems/DriveSystem.java

```diff
@@ -4,14 +4,89 @@
 
 package frc.robot.subsystems;
 
+import com.kauailabs.navx.frc.AHRS;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.util.sendable.Sendable;
+import edu.wpi.first.util.sendable.SendableBuilder;
+import edu.wpi.first.wpilibj.CAN;
+import edu.wpi.first.wpilibj.drive.MecanumDrive;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
-public class DriveSystem extends SubsystemBase {
+public class DriveSystem extends SubsystemBase implements Sendable {
```

Comment:
Ditch implements Sendable

Reply:
this is because SubsystemBase already implements Sendable, not because we are getting rid of Sendable